addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5412,6 +5413,9 @@ registry.layout_column = SnippetOptionWidget.extend(ColumnLayoutMixin, {
                     onlyImages: true,
                     save: imageEl => {
                         isImageSaved = true;
+                        imageLoadedPromise = new Promise(resolve => {
+                            imageEl.addEventListener("load", () => resolve(), {once: true});
+                        });
```

Comment:
It would make sense to remove the event listener once loaded no ?

Reply:
That's why I used `once` => it removes the listener after being called :slightly_smiling_face: (see [doc](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#once))

> A boolean value indicating that the listener should be invoked at most once after being added. If true, the listener would be automatically removed when invoked. If not specified, defaults to false.